_data/developer.yml

```diff
@@ -251,6 +251,14 @@ websites:
       software: Yes
       doc: https://www.mapbox.com/help/two-step-verification/
 
+      name: Messente
+      url: https://messente.com/
+      img: messente.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: https://messente.com/documentation/verification-api
```

Comment:
@jbaanus it seems like your `doc` link describes how to use 2fa api, not how to enable 2fa for the users of messente service.

Reply:
Updated the PR and removed the link to documentation.